.storybook/webapp-styles.less

```diff
@@ -0,0 +1,136 @@
+/**
+ * This file contains styles that are used in webapp that ultimately
```

Comment:
I'd like to avoid, if possible, references to projects outside of Perseus. I realize that might be pedantic, but Perseus is open source and although webapp is our primary consumer, I think it's worthwhile to stay at least somewhat agnostic of where Perseus is used.

Also, as stated in `story-wrapper.tsx`, I'm loathe to copy this much styling from an outside consumer just in Storybook. It gives us a false sense of what final styling will be in any given location if we've duplicated the styles only in Storybook, but not in our production styles.